### ISSUE TITLE
Migrate from Robby's personal repository to Oh My Zsh organization

### ISSUE THREAD

**Issue by robbyrussell** [ohmyzsh/ohmyzsh#8388]
After a decade of running this under @robbyrussell personal account...the project has grown enough that it's time to move this to a proper organization that @mcornella has setup.

https://github.com/ohmyzsh

We'll work on outlining a strategy for this to make sure this impacts next-to-zero current users. :-) 

* https://help.github.com/en/github/administering-a-repository/transferring-a-repository

### Tasks

- [x] 1. Update main / billing email address to Robby's email
- [x] 2. Notify the teams
- [x] 3.1. [Transfer repository](https://help.github.com/en/github/administering-a-repository/transferring-a-repository) to @ohmyzsh
- [x] 3.2. [Rename repository](https://help.github.com/en/github/administering-a-repository/renaming-a-repository) from oh-my-zsh to ohmyzsh for consistency
- [x] 4. Update all instances of old URL in codebase
- [x] 5. Update all instances of old URL in the wiki
- [x] 6. Update website install guide
- [x] 7. Add automatic remote reset when performing an upgrade (this should check the remote matches robbyrussell/oh-my-zsh)
- [ ] ... anything else?

**Comment by mcornella** [comment]
Items 4 and 7 are done in https://github.com/robbyrussell/oh-my-zsh/pull/8394. After we make the transfer we can merge it.

I've also replaced all instances of the old URL in a separate branch in the wiki, we just need to merge it once the transfer is done.

I've checked the "notify teams" checkbox, I don't know if there's anyone else we need to notify. Certainly it would be good to tweet it and maybe send an email via the newsletter, though the transfer should finish without problems.

**Comment by larson-carter** [comment]
@robbyrussell or @mcornella How can we go about updating the website?

**Comment by mcornella** [comment]
Robby will take care of that.

**Comment by larson-carter** [comment]
@mcornella We also need to introduce the Discord server.

**Comment by robbyrussell** [comment]
@mcornella would you like me to initiate the Github transfer tomorrow?

RE: 3.2 -- renaming the repo from 'oh-my-zsh' to 'ohmyzsh', 

I'm a little nervous about this particular one (as the repo name does match the config file and local directory within the user's machines). A few quick thoughts:

* Are we confident that Github will redirect that, too?
* Does this impact anything with how the installs/updates work with the naming of the local directory and config that gets created for users? 

Could we explore this particular item at some future point?

**Comment by larson-carter** [comment]
@robbyrussell 

Now that you mention GitHub redirecting the configs. I think that it would be best to ask GitHub directly pertain to that question just to make sure. 

I am fine with putting the rename on hold until a future date.

**Comment by mcornella** [comment]
> I'm a little nervous about this particular one (as the repo name does match the config file and local directory within the user's machines)

The config file and local directory are set up with an explicit name: [here's the definition of `$ZSH`](https://github.com/robbyrussell/oh-my-zsh/blob/26df8a22741ea4bb5264233344e42ce900fd26b6/tools/install.sh#L35) and here's where that is used [in the `git clone`](https://github.com/robbyrussell/oh-my-zsh/blob/26df8a22741ea4bb5264233344e42ce900fd26b6/tools/install.sh#L97) and [in the zshrc file](https://github.com/robbyrussell/oh-my-zsh/blob/26df8a22741ea4bb5264233344e42ce900fd26b6/tools/install.sh#L133-L135). Existing configurations won't notice the change.

Then, in existing repositories, we will [change the git remote to ohmyzsh/ohmyzsh](https://github.com/robbyrussell/oh-my-zsh/pull/8394/commits/cf90e4e6dc4d159e0f71aca813efbe407c8ca3e2#diff-6d2fa595805de6f141a64f905f4db641R37), but that is only ["to reduce confusion"](https://help.github.com/en/github/administering-a-repository/renaming-a-repository):

> In addition to redirecting web traffic, all git clone, git fetch, or git pushoperations targeting the previous location will continue to function as if made on the new location. **However, to reduce confusion, we strongly recommend updating any existing local clones to point to the new repository URL.**

As far as renaming the repository, the same caveats apply as in [transfering it](https://help.github.com/en/github/administering-a-repository/transferring-a-repository#whats-transferred-with-a-repository). All of these will continue to work:

- [x] Issues (including PRs)
- [x] Wiki
- [x] Stars
- [x] Watchers
- [x] Webhooks
- [x] Forks associated
- [x] Issue assignments to members of the organization

What's not redirected are GitHub Pages of the project, but we're not using them.

Also, I contacted GitHub support to see if we there was a 1-step process for transfer + rename, but there isn't one: 
![image](https://user-images.githubusercontent.com/1441704/69139852-a74c9000-0ac1-11ea-9510-71c1abd739ae.png)

Either way, there aren't any issues we need to be aware of. There is a Warning when [renaming a repository though](https://help.github.com/en/github/administering-a-repository/renaming-a-repository), which is
> Warning: If you create a new repository under your account with the same name as the transferred repository, existing redirects to the transferred repository will break. Instead, use a different name for the new repository.

meaning we can't have a repository named `oh-my-zsh` in the ohmyzsh org once the rename is done. Otherwise requests will go to this newly created repository instead of the ohmyzsh one.

Also, I tested the rename feature with [my fork](https://github.com/mcornella/ohmyzsh) and it looks to be working correctly:
![image](https://user-images.githubusercontent.com/1441704/69140203-55f0d080-0ac2-11ea-8261-21093c28a1b7.png)

**Comment by robbyrussell** [comment]
This is now moved to (and renamed) ohmyzsh...and the website has been updated!

![image](https://user-images.githubusercontent.com/257/69292571-591bb780-0bbb-11ea-9d2e-d522d4949910.png)

Yay!! This should have been done years ago.  🥇
